src/Microsoft.DotNet.Tools.Compiler.Native/Program.cs

```diff
@@ -32,6 +32,9 @@ private static int ExecuteApp(string[] args)
             try
             {
                 var cmdLineArgs = ArgumentsParser.Parse(args);
+
+                if (cmdLineArgs.IsHelp) return 0;
```

Comment:
Same comment here as above, but I think there might be more situations where this would be non-zero. 

Like if one of the required params was missing.